<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureForm</class>
 <widget class="QWidget" name="PictureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Picture</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>&amp;y(t)</string>
        </property>
        <property name="buddy">
         <cstring>y</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sLabel">
        <property name="text">
         <string>&amp;s(t)</string>
        </property>
        <property name="buddy">
         <cstring>s</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="file">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="fileAdd">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>&amp;x(t)</string>
        </property>
        <property name="buddy">
         <cstring>x</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>&amp;File</string>
        </property>
        <property name="buddy">
         <cstring>file</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="fileRemove">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="fileSend">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icons/export.svg</normaloff>:/icons/export.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="x">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QLineEdit" name="y">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QLineEdit" name="s">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
